docs/book/src/topics/ignition-support.md

```diff
@@ -109,6 +134,27 @@ spec:
     namePrefix: my-custom-secure-bucket-prefix-
 ```
 
+### Store Ignition config as PlainTextUserData
+
+WARNING: this is discouraged as is not considered as secure as other storage types.
```

Comment:
Would appreciate if this warning is highlighted more, like in bold/code blocked.

Reply:
Added it in a warning note block and set it as bold text.